pkgs/games/dwarf-fortress/game.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
How many of these versions do we need in nixpkgs? And what is the criteria to, which versions we add?

Reply:
We can probably delete some... Last 1-2 are usually compatible with all of the tools. Sometimes people play on older ones because new versions break save files.